### DISCUSSION TITLE
Deploying Orchestrator(v1.8) on AWS

### DISCUSSION THREAD

**Discussion by oloruntoba-madamori** [magma/magma#15302] [Q&A]
Hello, I'm trying to deploy the orchestrator on aws and I ran into the following errors.
This is my first time deploying orc8r and I'm following the instructions on the official magma docs page for v1.8

I was able to successfully terraform apply the infrastructure and the secrets as specified by the instructions in the docs.

Running into this error when I try to install the orchestrator application.
Need help!! thanks.

![image](https://github.com/magma/magma/assets/130354039/0edb9e0c-9462-46f3-b714-0b56c177de97)

**Comment by moyilaakhila** [comment]
Hi @oloruntoba-madamori ,
Can you please confirm which version of Kubernetes you are using?

**Reply by oloruntoba-madamori** [reply]
kubernetes/eks version 1.27

**Comment by mohinali** [comment]
Hi @oloruntoba-madamori,
The issue you are encountering indicates that you are trying to create PodSecurityPolicy, ClusterRole, and CronJob resources in your Kubernetes cluster. However, Kubernetes/EKS version 1.27 does not support these resources in the policy/v1beta1 API version.

To resolve this issue, follow these steps:

If your Kubernetes/EKS cluster is running version 1.27 or later and does not support the policy/v1beta1 API version for PodSecurityPolicy, ClusterRole, and CronJob, you will need to update your release manifests to use the policy/v1 API version instead. Here's how you can do it:

Open your deployment manifests for PodSecurityPolicy, ClusterRole, and CronJob resources.
Locate the sections where these resources are defined.
pdate the API version from policy/v1beta to policy/v1
Save your updated manifests.
Apply the modified manifests to your cluster:

By making this change, you will align your resources with the supported API version for Kubernetes/EKS version 1.27 and later. This should resolve the issue you are facing.

Thanks,
mohin.

**Reply by oloruntoba-madamori** [reply]
Apologies if this is a silly question, but where do I find the deployment manifests?
I'm new to this.

**Comment by jordanvrtanoski** [comment]
@ShubhamTatvamasi please take a look if we need to update the scripts in 1.9 to avoid this issue.

**Comment by panyogesh** [comment]
Will target it for 1.9 and verify once the branch is cut.
